### DISCUSSION TITLE
Async errors with new 4.0 version?

### DISCUSSION THREAD

**Discussion by thisismygitrepo** [zellij-org/zellij#3279] [Q&A]
I didn't want to raise an issue since I don't have a clear error log on this, but I have a complex program with async, multithreading multiprocessing ops and it failed since I updated, and it works if I revert to 0.39.2 I'm wondering if any one else is having similar issues and whether Zellij can be blamed at all.

**Comment by imsnif** [comment]
Is this a program you run inside Zellij or a plugin?

**Reply by thisismygitrepo** [reply]
A python program fired from shell in a zellij pane (not plugin), it launches large number of streams using websockets and save the stream on local database, and I'm getting async errors that I don't really understand. I'm back to 0.39.2 for now.

**Reply by imsnif** [reply]
While I am not discounting your experience - it is very unlikely this has anything to do with Zellij. Zellij does not handle or interfere with programs run inside shells that run inside panes.

More likely is that this has to do with the environment in which Zellij is run in one way or another.
